plugins/catalog-backend/src/ingestion/LocationAnalyzer.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Logger } from 'winston';
+import GitUriParser from 'git-url-parse';
```

Comment:
Since this is a function and not a class it should probably be lowercase. The existing convention in the code base is like https://github.com/backstage/backstage/blob/master/packages/backend-common/src/reading/BitbucketUrlReader.ts#L18

Reply:
But I've copied it from other place in the codebase:
https://github.com/backstage/backstage/blob/0c0798f082ae6f443798196d835d727837046477/plugins/scaffolder-backend/src/scaffolder/stages/prepare/github.ts#L23
(will fix with next commit)

Reply:
It looks like the scaffolder is the only spot not following the lowercase function convention used elsewhere :(

Reply:
This is one reason that we have https://github.com/backstage/backstage/blob/master/docs/architecture-decisions/adr003-avoid-default-exports.md :) If you search for `from 'git-url-parse'` you'll see that there are a bunch of different names chosen. But `parseGitUrl` is by far the most common and it makes sense to me to be a verb in the imperative form since it's a function that performs an action.